packages/opentelemetry-metrics/test/Meter.test.ts

```diff
@@ -381,6 +415,72 @@ describe('Meter', () => {
         assert.ok(gauge instanceof NoopMetric);
       });
     });
+
+    describe('.bind()', () => {
+      it('should create a measure handle', () => {
+        const measure = meter.createMeasure('name') as MeasureMetric;
+        const handle = measure.bind(labelSet);
```

Comment:
nit: update name `handle` to `bind`

Reply:
I think `boundMeasure` would make more sense

Reply:
@markwolff Could you please fix last round of comments and resolve the conflicts? Looks good to go after that.

Reply:
Rebased and addressed this comment 8e53f1e